mlir-tensorrt/tensorrt/test/Target/TensorRT/shape.mlir

```diff
@@ -12,6 +12,16 @@ func.func @tensorrt_shape_op(
 }
 
 
+// CHECK-LABEL: @tensorrt_shape_op_int64
+//  CHECK-SAME: tensorrt.engine
+func.func @tensorrt_shape_op_int64(
```

Comment:
I think this specific test would need to be predicated using "// REQUIRES: tensorrt-version-ge-10.0" in a separate file if one doesn't already exist for this op.

Reply:
Makes sense, I have moved the test: https://github.com/NVIDIA/TensorRT-Incubator/pull/563